### PR TITLE
Update access-control.yaml identus-admins team

### PR DESCRIPTION
**Pull request by yshyn-iohk**
Add members to the identus-admins team

### CODE REVIEW

**Review by ryjones — APPROVED** [review]

File: access-control.yaml [hunk 1]
```diff
@@ -949,6 +949,13 @@ teams:
   - name: identus-admins
     maintainers:
       - yshyn-iohk
```
Comment (mkbreuningIOHK) [inline access-control.yaml:951]: Is one maintainer enough? What is happening is the maintainer is not available?

File: access-control.yaml [hunk 2]
```diff
@@ -949,6 +949,13 @@ teams:
...
+    members:
+      - FabioPinheiro
+      - amagyar-iohk
+      - milosbackonja
+      - patlo-iog
+      - Dale-iohk
+      - elribonazo
   - name: identus-maintainers
     maintainers:
       - essbante-io
```
Comment (mkbreuningIOHK) [inline access-control.yaml:961]: Is one maintainer enough? What is happening is the maintainer is not available?

**Review by mkbreuningIOHK — COMMENTED** [review]
Left a comment on the risk that there is only one maintainer.

File: access-control.yaml (hunk 2 above)
Comment (ryjones) [reply access-control.yaml:961]: It exists as a fallback. CLOWarden is the maintainer overall, followed by LF staff.

**Pull request merged**